src/examples/install-specific-firefox-and-geckodriver.yml

```diff
@@ -0,0 +1,13 @@
+description: >
+  Example job that installs specific versions of Firefox and Geckodriver
+
+usage:
+  version: 2.1
+  orbs:
+      browser-tools: circleci/browser-tools@0.1.3
+  workflows:
+    main:
+      jobs:
+        - browser-tools/install-browser-tools:
```

Comment:
Again here, this is referencing a non-existant job. The job would need to be defined within this config and then utilize the commands from this orb.

Here is an example of another orb which only contains commands. The usage example shows the creations of a job which utilizes these commands. https://circleci.com/orbs/registry/orb/circleci/aws-cli#usage-install_aws_cli